microsoft/testsuites/network/sriov.py

```diff
@@ -694,10 +695,27 @@ def verify_irqbalance(self, environment: Environment) -> None:
         server_node = cast(RemoteNode, environment.nodes[0])
         client_node = cast(RemoteNode, environment.nodes[1])
 
-        server_node.tools[Sed].append(
-            "IRQBALANCE_ARGS='--debug'", "/etc/default/irqbalance", True
-        )
-        server_node.tools[Service].restart_service("irqbalance")
+        if (
+            server_node.execute(
+                "command -v irqbalance", shell=True, sudo=True
+            ).exit_code
+            != 0
+        ):
+            raise SkippedException("irqbalance is not installed")
+
+        # Get the irqbalance version if we can
+        if isinstance(server_node.os, Posix):
+            try:
+                environment.log.debug(
+                    "irqbalance version: "
+                    f"{server_node.os.get_package_information('irqbalance')}"
+                )
```

Comment:
can irqbalance --version give the version we want?

Reply:
For example, the original irqbalance issue was seen in `1.8.0+1build1` but not `1.8.0-1ubuntu0.1`. If we ran `irqbalance --version` they would both return 1.8.0.